doc/nrf/libraries/networking/multicell_location.rst

```diff
@@ -21,9 +21,18 @@ When the location service has resolved the location based on the cell measuremen
 After receiving the HTTP response, the Multicell location library parses the response and returns the location to the caller.
 
 The library supports location services from `nRF Cloud Location Services`_, `HERE Positioning`_ (v1 and v2) and `Skyhook Precision Location`_.
-Note that nRF Cloud currently is a single-cell location service, and does not make use of neighboring cells in location resolution.
 To use the location services, see the respective documentation for account setup and for getting the required credentials for authentication.
 
+.. reprovision_cert_note_start
+
+.. note::
+
+   nRF9160 DK and Thingy:91 devices are shipped with RSA256 certificates.
+   To use the Multicell location library with nRF Cloud, you must delete the device from nRF Cloud and reprovision it with a new ES256 device certificate.
```

Comment:
Another option is to use [https://api.nrfcloud.com/v1#operation/RegisterPublicKeys](https://api.nrfcloud.com/v1#operation/RegisterPublicKeys), see use case 2.
Not sure if this is "easier" for the user or not.  They wouldn't need to delete and re-add their device to the cloud.  They _would_ need to either generate a private key using [KEYGEN](https://docs.nrfcloud.com/Guides/GettingStarted/Devices#securely-generating-credentials-on-the-nrf9160) or generate their own private key and write it to their device.  This would be done in a different modem sec tag and `CONFIG_MULTICELL_LOCATION_NRF_CLOUD_JWT_SEC_TAG` would need to be updated accordingly.